components/ingest-service/cmd/ingest-service/commands/serve.go

```diff
@@ -94,7 +94,7 @@ func readCliParams() *serveropts.Opts {
 		PGDatabase:                    viper.GetString("postgresql-database"),
 		AuthzAddress:                  viper.GetString("authz-address"),
 		CerealAddress:                 viper.GetString("cereal-address"),
-		EventAddress:                  viper.GetString("event-address"),
```

Comment:
We do not need the event-service in the ingest-service since the project update moved to use cereal for connecting all the services together.